src/components/MapView/index.tsx

```diff
@@ -457,7 +457,7 @@ function MapView({ classes }: MapViewProps) {
           });
         })}
         {/* These are custom layers which provide functionality and are not really controllable via JSON */}
-        <AnalysisLayer before={firstSymbolId} />
+        <AnalysisLayer before={boundaryLayer} />
```

Comment:
I think `before` should still be a string, and you would pass `layer-${boundaryLayer.id}-line`

Reply:
@hav142 yes I think it would be nice to push towards uniformization. For example we could have:
- all layers accept a before parameter
- control ordering from`MapView/index.tsx` so that it is clearer and easier to update in the future.

What do you think?

Reply:
I agree that we should control the ordering from one place and use `before` parameter to easier maintenance. I have added it to the other layer types, but for `AdminLevelDataLayer` it seems have unique boundary and it own setting. should we just leave it like that or we can replace it as well?

Reply:
Have you tested that it works properly? If it does, then we can leave it like that for now @hav142

Reply:
I have tested other layer types and they work, but I will do more detailed tests in other countries too.

Reply:
Make sure to try `AdminLevelDataLayer` and once you confirm it works I think we are good to go